src/Compute/Compute/VirtualMachine/Operation/UpdateAzureVMCommand.cs

```diff
@@ -182,6 +182,18 @@ public class UpdateAzureVMCommand : VirtualMachineBaseCmdlet
            Mandatory = false)]
         [ValidateNotNullOrEmpty]
         public bool? EnableSecureBoot { get; set; } = null;
+        
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The ETag of the transformation. Omit this value to always overwrite the current resource. Specify the last-seen ETag value to prevent accidentally overwriting concurrent changes.")]
```

Comment:
Is this the etag or ifmatch? HelpMessage needs to be more clear.

Reply:
this is ifMatch from swagger

Reply:
@grizzlytheodore , even so, I think it needs to improved, or we need to understand it better bc I am still confused by this. Maybe the swagger description needs to be updated too.

Reply:
@Sandido  improved, as requested !